tests/unit/cirrus/build_tests.py

```diff
@@ -50,17 +50,22 @@ def setUp(self):
         self.os_cwd_patcher = mock.patch('cirrus.build.os.getcwd')
         self.pypi_auth_patcher = mock.patch('cirrus.build.get_pypi_auth')
         self.cirrus_home_patcher = mock.patch('cirrus.build.cirrus_home')
+        self.pypirc_patcher = mock.patch('cirrus.build.PypircFile')
 
         self.build_params = {}
         self.pypi_url_value = None
 
+        self.mock_pypirc_inst = mock.Mock()
+        self.mock_pypirc_inst.index_servers = []
```

Comment:
No, because i'm not mocking the property, i'm replacing the whole class with a mock. If I was using a PypircFile instance and mocking its attrs/methods, then I would need that I think.